website/pages/docs/configuration/telemetry.mdx

```diff
@@ -37,6 +37,22 @@ The following options are available on all telemetry configurations.
 - `enable_hostname_label` `(bool: false)` - Specifies if all metric values should
   contain the `host` label with the local hostname. It is recommended to enable
   `disable_hostname` if this option is used.
+- `lease_metrics_epsilon` `(string: "1hr")` - Specifies the time difference between collections
+  of leases by expiry time. For example, for the default value of 1 hour, the `vault.expire.leases.by_expiration`
+  metric will aggregate the total number of expiring leases for 1 hour buckets, starting from the current time.
+  Note that leases are put into buckets by rounding. For example, if `lease_metrics_epsilon` is set to 1hr and 
+  lease A expires 25 minutes from now, and lease B expires 35 minutes from now, then lease A will be in a
+  different bucket from lease B. 
+- `num_lease_metrics_buckets` `(int: 168)` - The number of expiry buckets for leases. For the default value for
+  example, 168 value labels for the `vault.expire.leases.by_expiration` metric will be reported, where each value 
+  has a time difference from the previous value that is specified by the `lease_metrics_epsilon` parameter. For the
```

Comment:
"has a time different from the previous value that is specified by the lease_metrics_epsilon parameter" 

=>

"each bucket is separated in time by the lease_metrics_epsilon parameter"?